Simplenote/Classes/SPEditorTextView.m

```diff
@@ -70,7 +69,11 @@ - (instancetype)init
                forKeyPath:@"contentOffset"
                   options:NSKeyValueObservingOptionNew
                   context:NULL];
-        
+        [self addObserver:self
+               forKeyPath:@"contentInset"
```

Comment:
Maybe constants for some of these strings that are being used in multiple places?